### PR TITLE
MAIN Add default value to input

### PR DESCRIPTION
**Pull request by herman-rogers**
- Add default value for inputs
- Fix errors with missing validation context

### CODE REVIEW

File: stories/Input.story.jsx [hunk 1]
```diff
@@ -23,14 +23,14 @@ storiesOf("Input", module)
     ))
   )
   .add(
-    "optional",
+    "default value",
```
Comment (lanwen) [inline stories/Input.story.jsx]: I think better to add also optional example without default value

Comment (herman-rogers) [reply stories/Input.story.jsx]: optional doesn't do anything afaik. It isn't even a prop on the input

Comment (herman-rogers) [reply stories/Input.story.jsx]: And the "basic" story, shows an input without any default values.

Comment (lanwen) [reply stories/Input.story.jsx]: anyway, better to have example with absent default value

**Review by lanwen — APPROVED** [review]

**Pull request merged**